From 47412ddb26acf227ee6cb6950f6e9ded01f3375c Mon Sep 17 00:00:00 2001
From: Jiachen Yang <farseerfc@gmail.com>
Date: Tue, 5 Jul 2016 21:10:07 +0900
Subject: [PATCH] add plugin definition in qmldir

Add `plugin AsemanToolsQml` in qmldir, otherwise cutegram will run into errors saying things defined in the plugin are not  types.
---
 qml/AsemanTools/qmldir | 1 +
 1 file changed, 1 insertion(+)

diff --git a/qml/AsemanTools/qmldir b/qml/AsemanTools/qmldir
index 391531a..3f41c5f 100755
--- a/qml/AsemanTools/qmldir
+++ b/qml/AsemanTools/qmldir
@@ -1,4 +1,5 @@
 module AsemanTools
+plugin AsemanToolsQml
 classname AsemanToolsPlugin
 typeinfo plugins.qmltypes
 AnimationItem 1.0 AnimationItem.qml
-- 
2.9.2

